app/constants.js

```diff
@@ -4,6 +4,7 @@ const ROLES = {
     AI: "Ai"
 };
 const ANONY_COOKIE_DURATION_MS = 5*60*1000; // 5 mins
+const KNOWN_USER_COOKIE_DURATION_MS = 12*60*60*1000 // 12 hrs 
```

Comment:
It looks like `ANONY_COOKIE_DURATION_MS` is used when making new cookies when starting a Game. We'll probably be reworking that later, anyways, so it doesn't matter now

Reply:
Anonymous users will never be logged in for hours. The cookie should only be made when they are in a game. It should only be used for the duration of the game (4 mins) I made it 5 just for extra safety. I don't wanna track an anonymous user unless we absolutely have to. Since they don't have an account I figured they shouldn't have a cookie lasting for a long time. They'll just generate a new token each time so on our end we will treat them as an anonymous websocket connection. 
If we kept the cookie for longer, we could track them for longer and tell whenever that "anonymous" user connects, which in that case they are no longer anonymous. 

The known user one, I made it 12 hours since they probably don't wanna log out and log in over and over